Octokit.Tests.Integration/Reactive/ObservableCommitStatusClientTests.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Reactive.Linq;
+using System.Threading.Tasks;
+using Octokit.Reactive;
+using Xunit;
+
+namespace Octokit.Tests.Integration.Reactive
+{
+    public class ObservableCommitStatusClientTests
+    {
+        public class TheGetAllMethod
+        {
+            readonly ObservableCommitStatusClient _commitStatusClient;
+            const string owner = "octokit";
+            const string name = "octokit.net";
+            const string reference = "master";
```

Comment:
Given `master` will actually change over time, we'd need to be careful here about what we're testing.

For example, you could lock this to 1335f37c6cde55de09c086cda283e70e0dfc339e which has 5 statuses and won't change. This means the first three tests will pass currently.